### PR TITLE
fix: merge coverage report

### PR DESCRIPTION
**Pull request by FeelyChau**

### CODE REVIEW

File: packages/daemon/package.json [hunk 1]
```diff
@@ -52,7 +53,7 @@
     "clear": "midway-bin clean",
     "dev": "cross-env NODE_ENV=local midway-bin dev --port=6927 --ts",
     "debug": "cross-env NODE_ENV=local midway-bin debug --port=6927 --ts",
-    "cov": "midway-bin cov --ts",
+    "cov": "midway-bin cov --ts --nyc='-r lcov -r text-summary'",
```
Comment (yorkie) [inline packages/daemon/package.json:56]: Why do we use text-summary, it causes us to lose some information. See https://istanbul.js.org/docs/advanced/alternative-reporters/#text-summary.

Comment (FeelyChau) [reply packages/daemon/package.json:56]: `-r lcov -r text-summary` means nyc will generate file lcov.info on disk and text-summary on console.

File: packages/costa/.nycrc [hunk 2]
```diff
@@ -22,5 +24,4 @@
   ],
   "all": true,
   "instrument": true
-}
-
+}
```
Comment (yorkie) [inline packages/costa/.nycrc:27]: EOF

File: packages/boa/package.json [hunk 3]
```diff
@@ -10,7 +10,8 @@
     "test": "npm run test:js && npm run test:ts",
     "test:js": "tape ./tests/**/*.js",
     "test:ts": "ts-node ./node_modules/.bin/tape ./tests/**/*.ts",
-    "cov": "npx nyc --reporter=lcov --reporter=text-summary npm run test",
+    "cov": "nyc --reporter=text-summary npm run test",
+    "cov-report": "nyc report -r=lcov",
```
Comment (yorkie) [inline packages/boa/package.json:14]:
```suggestion
    "cov:report": "nyc report -r=lcov",
```

File: packages/core/package.json [hunk 4]
```diff
@@ -13,7 +13,8 @@
     "clean": "rm -rf ./dist && rm -rf tsconfig.tsbuildinfo",
     "dev-compile": "tsc -b tsconfig.json",
     "compile": "tsc -b tsconfig.json",
-    "cov": "npx nyc --reporter=lcov --reporter=text-summary npm run test"
+    "cov": "nyc -r text-summary npm run test",
+    "cov-report": "nyc report -r lcov"
```
Comment (yorkie) [inline packages/core/package.json:17]: Ditto

**Review by yorkie — CHANGES_REQUESTED** [review]

File: packages/daemon/test/runner/helper.test.ts [hunk 5]
```diff
@@ -4,6 +4,8 @@ import * as helper from '../../src/utils';
 import * as sinon from 'sinon';
 import * as fs from 'fs-extra';
 import { join } from 'path';
+import * as request from 'request-promise';
+const Bluebird = require( 'bluebird');
```
Comment (yorkie) [inline packages/daemon/test/runner/helper.test.ts:8]: Shall we use `Promise` instead?

**Review by yorkie — CHANGES_REQUESTED** [review]

File: packages/daemon/test/runner/helper.test.ts (hunk 5 above)
Comment (FeelyChau) [reply packages/daemon/test/runner/helper.test.ts]: Theoretically, `Bluebird` is compatible with `Promise`, but the data types are different, so we should mark the Promise object as any.
Updated.

Comment (yorkie) [reply packages/daemon/test/runner/helper.test.ts]: I can't get that why do we mark the `Promise` object as `any`?

Comment (FeelyChau) [reply packages/daemon/test/runner/helper.test.ts]: `Bulebird` is not completely the same with `Promise`, it will fail when check data type by typescript.

**Review by yorkie — APPROVED** [review]

**Pull request merged**